ui/hooks/src/use-notifications.ts

```diff
@@ -57,7 +59,7 @@ export function useFetchNotifications(loggedEthAddress: string) {
 
 /**
  * Hook to mark a notification as read
- * pass the messageId to the mutate function
+ * pass the messageId to the onMutate function
```

Comment:
to the `mutate` function. NOT to the `onMutate`. You never call `hook.onMutate()`